sktime/forecasting/ttm.py

```diff
@@ -218,6 +218,7 @@ def _fit(self, y, X, fh):
         -------
         self : reference to self
         """
+        raise ValueError("test_error")
```

Comment:
Why did you add this error?

Reply:
I recommended so, for checking whether the estimator is actually being tested, something which @julian-fong had reason to doubt.

I also recommended to explain these things in the GitHub pull request, or other people (like you) might end up confused...